component/main.jsonnet

```
@@ -34,7 +40,9 @@ local common_outputs = {
   cluster_dns: cluster_dns[params.provider],
 };
 local outputs = {
-  cloudscale: common_outputs,
+  cloudscale: common_outputs {
+    hieradata_mr: '${module.cluster.hieradata_mr}',
+  },
   exoscale: common_outputs {
     hieradata_mr: '${module.cluster.hieradata_mr}',
   },
```

Move `hieradata_mr` to `common_outputs` and simplify this to

```suggestion
  cloudscale: common_outputs,
  exoscale: common_outputs,
```